src/components/CodeBlock/CodeBlock.tsx

```diff
@@ -5,23 +5,18 @@ import { monokaiSublime } from "react-syntax-highlighter/dist/esm/styles/hljs";
 import * as styling from "./CodeBlock.module.css";
 
 type CodeBlockProps = {
-  index: number;
   handleStyleChange: (
-    index: number,
+    id: string,
     key: keyof CustomStyle,
     value: CustomStyle[keyof CustomStyle],
   ) => void;
   style: CustomStyle;
 };
 
-export const CodeBlock = ({
-  style,
-  index,
-  handleStyleChange,
-}: CodeBlockProps) => {
+export const CodeBlock = ({ style, handleStyleChange }: CodeBlockProps) => {
   return (
     <div className={styling.codeBlock}>
-      <FormLabel htmlFor="css">CSS</FormLabel>
+      <FormLabel htmlFor="css">CSS shown in the text editor</FormLabel>
```

Comment:
```suggestion
      <FormLabel htmlFor="css">CSS to apply in Structured Text editor</FormLabel>
```

You're not showing css, you're applying it to style something in the text editor